source/fundamentals/connection/connect.txt

```diff
@@ -61,146 +76,140 @@ a different dbpath, but all using the same replica set name "foo".
 
   $ mongod --port 27019 --dbpath /data/db2 --replSet foo
 
-Initializing the Set
-~~~~~~~~~~~~~~~~~~~~
+Your nodes are now running.
+
+Initializing and Connecting to the Replica Set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-At this point all of our nodes are up and running, but the set has yet
-to be initialized. Until the set is initialized no node will become
-the primary, and things are essentially "offline".
+Before you can use your nodes, you must initialize your replica set and set one
+node as the primary node. Note that the following process is specific to uninitialized
+replica sets.
 
-To initialize the set we need to connect directly to a single node and run the
+To initialize the set, connect directly to a single node and run the
 initiate command using the ``directConnection`` option:
 
 .. code-block:: python
 
-  >>> from pymongo import MongoClient
-  >>> c = MongoClient('localhost', 27017, directConnection=True)
+   >>> from pymongo import MongoClient
+   >>> c = MongoClient('localhost', 27017, directConnection=True)
 
-.. note:: We could have connected to any of the other nodes instead,
-   but only the node we initiate from is allowed to contain any
+.. note:: You can connect to any of the node you created,
+   but only the node you initiate (in this case, the node running on port 27017) from is allowed to contain
```

Comment:
Q: Is "initiate...from" the right syntax?

Reply:
Changed to "connect through". Also tweaked because I think it was still kind of awkward.